packages/client/hmi-client/src/page/Home.vue

```diff
@@ -134,12 +134,13 @@
 				v-if="isNewProjectModalVisible"
 				class="modal"
 				@modal-mask-clicked="isNewProjectModalVisible = false"
+				@modal-enter-press="createNewProject"
 			>
 				<template #header>
 					<h4>Create project</h4>
 				</template>
 				<template #default>
-					<form>
+					<form @submit.prevent>
```

Comment:
when there is a form element - i have prevent the default behavior that which refreshes the page